java/test/jmri/jmrit/display/layoutEditor/TrackSegmentTest.java

```diff
@@ -16,26 +16,71 @@
  */
 public class TrackSegmentTest {
 
+    static private LayoutEditor layoutEditor = null;
+    static private TrackSegment trackSegment = null;
```

Comment:
You really want both the TrackSegment and LayoutEditor to be created every time a test runs to make sure one test doesn’t contaminate the next.  You need to remove the static here.